### ISSUE TITLE
L: nan C: nan S: nan

### ISSUE THREAD

**Issue by rw1995** [dlyldxwl/fssd.pytorch#8]
when i train VOC2007，it work ,but the progress is terrible
`Epoch:1 || epochiter: 0/1034|| Totel iter 0 || L: 201.1674 C: 330.5661 S: 531.7335||Batch time: 12.2342 ||LR: 0.0010000
Epoch:1 || epochiter: 10/1034|| Totel iter 10 || L: nan C: nan S: nan||Batch time: 0.8318 ||LR: 0.0010000
Epoch:1 || epochiter: 20/1034|| Totel iter 20 || L: nan C: nan S: nan||Batch time: 0.8154 ||LR: 0.0010000
Epoch:1 || epochiter: 30/1034|| Totel iter 30 || L: nan C: nan S: nan||Batch time: 0.8430 ||LR: 0.0010000
Epoch:1 || epochiter: 40/1034|| Totel iter 40 || L: nan C: nan S: nan||Batch time: 0.8290 ||LR: 0.0010000
Epoch:1 || epochiter: 50/1034|| Totel iter 50 || L: nan C: nan S: nan||Batch time: 0.8428 ||LR: 0.0010000
Epoch:1 || epochiter: 60/1034|| Totel iter 60 || L: nan C: nan S: nan||Batch time: 0.8450 ||LR: 0.0010000
Epoch:1 || epochiter: 70/1034|| Totel iter 70 || L: nan C: nan S: nan||Batch time: 0.8438 ||LR: 0.0010000
Epoch:1 || epochiter: 80/1034|| Totel iter 80 || L: nan C: nan S: nan||Batch time: 0.8202 ||LR: 0.0010000
Epoch:1 || epochiter: 90/1034|| Totel iter 90 || L: nan C: nan S: nan||Batch time: 0.8445 ||LR: 0.0010000
Epoch:1 || epochiter: 100/1034|| Totel iter 100 || L: nan C: nan S: nan||Batch time: 0.8235 ||LR: 0.0010000
Epoch:1 || epochiter: 110/1034|| Totel iter 110 || L: nan C: nan S: nan||Batch time: 0.8498 ||LR: 0.0010000
Epoch:1 || epochiter: 120/1034|| Totel iter 120 || L: nan C: nan S: nan||Batch time: 0.8209 ||LR: 0.0010000
Epoch:1 || epochiter: 130/1034|| Totel iter 130 || L: nan C: nan S: nan||Batch time: 0.8657 ||LR: 0.0010000
Epoch:1 || epochiter: 140/1034|| Totel iter 140 || L: nan C: nan S: nan||Batch time: 0.8319 ||LR: 0.0010000
Epoch:1 || epochiter: 150/1034|| Totel iter 150 || L: nan C: nan S: nan||Batch time: 0.8366 ||LR: 0.0010000
Epoch:1 || epochiter: 160/1034|| Totel iter 160 || L: nan C: nan S: nan||Batch time: 0.8760 ||LR: 0.0010000
Epoch:1 || epochiter: 170/1034|| Totel iter 170 || L: nan C: nan S: nan||Batch time: 0.8323 ||LR: 0.0010000
Epoch:1 || epochiter: 180/1034|| Totel iter 180 || L: nan C: nan S: nan||Batch time: 0.8389 ||LR: 0.0010000
Epoch:1 || epochiter: 190/1034|| Totel iter 190 || L: nan C: nan S: nan||Batch time: 0.8319 ||LR: 0.0010000
Epoch:1 || epochiter: 200/1034|| Totel iter 200 || L: nan C: nan S: nan||Batch time: 0.8414 ||LR: 0.0010000
Epoch:1 || epochiter: 210/1034|| Totel iter 210 || L: nan C: nan S: nan||Batch time: 0.8519 ||LR: 0.0010000
`
could you help me?

**Comment by rw1995** [comment]
`
Epoch:1 || epochiter: 0/1034|| Totel iter 0 || L: 201.1674 C: 330.5661 S: 531.7335||Batch time: 12.2342 ||LR: 0.0010000
Epoch:1 || epochiter: 10/1034|| Totel iter 10 || L: nan C: nan S: nan||Batch time: 0.8318 ||LR: 0.0010000
Epoch:1 || epochiter: 20/1034|| Totel iter 20 || L: nan C: nan S: nan||Batch time: 0.8154 ||LR: 0.0010000
Epoch:1 || epochiter: 30/1034|| Totel iter 30 || L: nan C: nan S: nan||Batch time: 0.8430 ||LR: 0.0010000
Epoch:1 || epochiter: 40/1034|| Totel iter 40 || L: nan C: nan S: nan||Batch time: 0.8290 ||LR: 0.0010000
Epoch:1 || epochiter: 50/1034|| Totel iter 50 || L: nan C: nan S: nan||Batch time: 0.8428 ||LR: 0.0010000
Epoch:1 || epochiter: 60/1034|| Totel iter 60 || L: nan C: nan S: nan||Batch time: 0.8450 ||LR: 0.0010000
Epoch:1 || epochiter: 70/1034|| Totel iter 70 || L: nan C: nan S: nan||Batch time: 0.8438 ||LR: 0.0010000
Epoch:1 || epochiter: 80/1034|| Totel iter 80 || L: nan C: nan S: nan||Batch time: 0.8202 ||LR: 0.0010000
Epoch:1 || epochiter: 90/1034|| Totel iter 90 || L: nan C: nan S: nan||Batch time: 0.8445 ||LR: 0.0010000
Epoch:1 || epochiter: 100/1034|| Totel iter 100 || L: nan C: nan S: nan||Batch time: 0.8235 ||LR: 0.0010000
Epoch:1 || epochiter: 110/1034|| Totel iter 110 || L: nan C: nan S: nan||Batch time: 0.8498 ||LR: 0.0010000
Epoch:1 || epochiter: 120/1034|| Totel iter 120 || L: nan C: nan S: nan||Batch time: 0.8209 ||LR: 0.0010000
Epoch:1 || epochiter: 130/1034|| Totel iter 130 || L: nan C: nan S: nan||Batch time: 0.8657 ||LR: 0.0010000
Epoch:1 || epochiter: 140/1034|| Totel iter 140 || L: nan C: nan S: nan||Batch time: 0.8319 ||LR: 0.0010000
Epoch:1 || epochiter: 150/1034|| Totel iter 150 || L: nan C: nan S: nan||Batch time: 0.8366 ||LR: 0.0010000
Epoch:1 || epochiter: 160/1034|| Totel iter 160 || L: nan C: nan S: nan||Batch time: 0.8760 ||LR: 0.0010000
Epoch:1 || epochiter: 170/1034|| Totel iter 170 || L: nan C: nan S: nan||Batch time: 0.8323 ||LR: 0.0010000
Epoch:1 || epochiter: 180/1034|| Totel iter 180 || L: nan C: nan S: nan||Batch time: 0.8389 ||LR: 0.0010000
Epoch:1 || epochiter: 190/1034|| Totel iter 190 || L: nan C: nan S: nan||Batch time: 0.8319 ||LR: 0.0010000
Epoch:1 || epochiter: 200/1034|| Totel iter 200 || L: nan C: nan S: nan||Batch time: 0.8414 ||LR: 0.0010000
Epoch:1 || epochiter: 210/1034|| Totel iter 210 || L: nan C: nan S: nan||Batch time: 0.8519 ||LR: 0.0010000

`

**Comment by nepulhl** [comment]
you can try to reduce the learning rate

**Comment by rw1995** [comment]
AMAZING~,NOW it work well,and I will train my own dataset
